### PR TITLE
Gf account page cleanup

### PR DESCRIPTION
**Pull request by frechg**
* Update account layout to fit with the rest of the app UI
* Add bulk pricing message
* Show active paid repo usage

screen shot 2015-03-09 at 10 31 11 am

### CODE REVIEW

File: app/views/accounts/show.haml [hunk 1]
```diff
@@ -3,47 +3,66 @@
...
-        - @account_page.repos.each do |repo|
-          %li.repo
-            %span.repo-name{ "ng-class" => "{private: #{repo.private}}" }
-              = repo.full_github_name
-    - else
-      %p.empty-state-message
-        You have no private repos billed to you
+    %article.active-private-repos
+      - if @account_page.repos.any?
+        %h3 Active Paid Repos
```
Comment (salbertson) [inline app/views/accounts/show.haml]: What do you think about "Active Private Repos" instead?

File: app/models/repo.rb [hunk 2]
```diff
@@ -59,6 +59,10 @@ def exempt?
     ENV["EXEMPT_ORGS"] && ENV["EXEMPT_ORGS"].split(",").include?(organization)
   end
 
+  def total_violations
+    Violation.joins(:build).where(builds: { repo_id: self.id }).length
```
Comment (salbertson) [inline app/models/repo.rb]: Redundant `self`?

Comment (salbertson) [inline app/models/repo.rb]: Should `#count` be used here instead of `#length`?

File: app/views/accounts/show.haml [hunk 3]
```diff
@@ -3,47 +3,66 @@
 = render 'settings'
 
 %section.account-section
-  %h2.label-text
-    Your monthly cost
+  %aside.account-nav
+    %p.message
+      If you'd like to pay for Hound in bulk on a monthly or annual
+      basis we're happy to make that happen. Email
+      #{mail_to "hound@thoughtbot.com"}
```
Comment (salbertson) [inline app/views/accounts/show.haml]: What do you think about moving the support email address to a translation so it's easy to change?

**Comment by salbertson** [comment]
I like it; I really like the review and violation numbers for each repo, it hopefully helps show the value of Hound and that it's doing something worth paying for.

I have a couple questions, looks good. :+1:

File: spec/features/account_spec.rb [hunk 4]
```diff
@@ -42,4 +42,20 @@
...
+  scenario "user sees paid repo usage" do
+    user = create(:user)
+    paid_repo = create(:repo, users: [user])
+    paid_repo.builds << create(:build, :failed)
+    paid_repo.builds << create(:build, :failed)
+    paid_repo.builds << create(:build)
+    create(:subscription, repo: paid_repo, user: user)
+
+    sign_in_as(user)
+
```
Comment (gylaz) [inline spec/features/account_spec.rb]: This newline is unnecessary.

**Pull request closed without merging**